build.gradle.kts

```diff
@@ -104,100 +104,39 @@ allprojects {
             force(
                 io.spine.internal.dependency.Grpc.protobufPlugin,
                 spine.base,
+                spine.testlib,
                 spine.server
             )
         }
     }
 }
 
-// Temporarily use this version, since 3.21.x is known to provide
-// a broken `protoc-gen-js` artifact and Kotlin code without access modifiers.
-// See https://github.com/protocolbuffers/protobuf-javascript/issues/127.
-//     https://github.com/protocolbuffers/protobuf/issues/10593
-val protocArtifact = "com.google.protobuf:protoc:3.19.6"
+object BuildSettings {
+    const val javaVersion = 11
 
-subprojects {
-    apply {
-        plugin("kotlin")
-        plugin("net.ltgt.errorprone")
-        plugin(Dokka.GradlePlugin.id)
-        plugin(Protobuf.GradlePlugin.id)
-    }
-
-    LicenseReporter.generateReportIn(project)
-
-    dependencies {
-        ErrorProne.apply {
-            errorprone(core)
-        }
-        testImplementation(spine.coreJava.testUtilServer)
-        testImplementation(kotlin("test-junit5"))
-        Truth.libs.forEach { testImplementation(it) }
-        testRuntimeOnly(JUnit.runner)
-    }
-
-    configurations.all {
-        resolutionStrategy {
-            force(
-                io.spine.internal.dependency.Protobuf.compiler,
-            )
-        }
-    }
-
-    tasks.test {
-        useJUnitPlatform()
-
-        testLogging {
-            events = setOf(PASSED, FAILED, SKIPPED)
-            showExceptions = true
-            showCauses = true
-        }
-    }
-
-    val javaVersion = JavaVersion.VERSION_11.toString()
```

Comment:
I would still use this expression inside `BuildSettings`. That seems less error-prone. 

Or... (con'd below)